app/src/main/java/com/xmartlabs/scasas/criminalintent/controller/CrimeController.java

```diff
@@ -19,18 +28,42 @@ public static CrimeController getInstance() {
     return INSTANCE;
   }
 
-  public void insertCrime(@NonNull Crime crime) {
-    crime.insert();
+  public Single<List<Crime>> getCrimesFromDataBase() {
+    return Single.fromCallable(() -> SQLite.select()
```

Comment:
Fix indentation. Lets try to whenever we read the code, know that calls from one object are all "in the same step of the stair", and methods that are called on a different one, are on a different "step"